### PR TITLE
importccl: add ignore lines option to DELIMITED

### PR DESCRIPTION
**Pull request by spaskob**
This is indeed when an input has a header that needs to be
ignored from importing. The new option is used as:
`IMPORT ... DELIMITED DATA ... WITH skip = 'n';`
where n is the number of lines to be ignored at beginning of file.

Release note (sql): add new option skip to DELIMITED.

### CODE REVIEW

File: pkg/ccl/importccl/import_stmt.go [hunk 1]
```diff
@@ -52,10 +52,11 @@ const (
...
 
-	mysqlOutfileRowSep   = "rows_terminated_by"
-	mysqlOutfileFieldSep = "fields_terminated_by"
-	mysqlOutfileEnclose  = "fields_enclosed_by"
-	mysqlOutfileEscape   = "fields_escaped_by"
+	mysqlOutfileRowSep      = "rows_terminated_by"
+	mysqlOutfileFieldSep    = "fields_terminated_by"
+	mysqlOutfileEnclose     = "fields_enclosed_by"
+	mysqlOutfileEscape      = "fields_escaped_by"
+	mysqlOutfileIgnoreLines = "ignore_lines"
```
Comment (dt) [inline pkg/ccl/importccl/import_stmt.go]: can we not use the existing `skip` option?

Comment (spaskob) [reply pkg/ccl/importccl/import_stmt.go]: done

**Review by dt — APPROVED** [review]

**Pull request merged**